neo.UnitTests/UT_FifoSet.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.IO.Caching;
+using System.Linq;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_FifoSet
+    {
+        [TestMethod]
+        public void FifoSetTest()
+        {
+            var a = UInt256.Zero;
+            var b = new UInt256();
```

Comment:
UInt256.Zero? Better than `new UInt256()`, no?

Reply:
I want the same data, with different objects

Reply:
Better to use two `new UInt256()`. Isn't it clearer?

Reply:
Yes, Is the same